collectors/ebpf.plugin/README.md

```diff
@@ -250,3 +250,85 @@ Our [initial testing](https://github.com/netdata/netdata/issues/8195) shows the
 nearly identical to our [apps.plugin collector](/collectors/apps.plugin/README.md), despite collecting and displaying
 much more sophisticated metrics. You can now use the eBPF to gather deeper insights without affecting the performance of
 your complex applications at any load.
+
+## SELinux
+
+When [SELinux](https://www.redhat.com/en/topics/linux/what-is-selinux) is enabled, it may prevent `ebpf.plugin` from
+starting correctly. Check the Agent's `error.log` file for errors like the ones below:
+
+```bash
+2020-06-14 15:32:08: ebpf.plugin ERROR : EBPF PROCESS : Cannot load program: /usr/libexec/netdata/plugins.d/pnetdata_ebpf_process.3.10.0.o (errno 13, Permission denied)
+2020-06-14 15:32:19: netdata ERROR : PLUGINSD[ebpf] : read failed: end of file (errno 9, Bad file descriptor)
+```
+
+You can also check for errors related to `ebpf.plugin` inside `/var/log/audit/audit.log`:
+
+```bash
+type=AVC msg=audit(1586260134.952:97): avc:  denied  { map_create } for  pid=1387 comm="ebpf_process.pl" scontext=system_u:system_r:unconfined_service_t:s0 tcontext=system_u:system_r:unconfined_service_t:s0 tclass=bpf permissive=0
+type=SYSCALL msg=audit(1586260134.952:97): arch=c000003e syscall=321 success=no exit=-13 a0=0 a1=7ffe6b36f000 a2=70 a3=0 items=0 ppid=1135 pid=1387 auid=4294967295 uid=994 gid=990 euid=0 suid=0 fsuid=0 egid=990 sgid=990 fsgid=990 tty=(none) ses=4294967295 comm="ebpf_proc
+ess.pl" exe="/usr/libexec/netdata/plugins.d/ebpf_process.plugin" subj=system_u:system_r:unconfined_service_t:s0 key=(null)
```

Comment:
Thiago is this `ebpf_process.plugin` or `ebpf.plugin`? I'm confused with the naming at this point.

Reply:
this is because I did not clean my `audit.log` when I was generating the rules, I fixed it now. Thank you to call attention for this.